--- performous-1.0/game/songparser.hh~	2015-12-05 14:07:56.354059559 +0100
+++ performous-1.0/game/songparser.hh	2015-12-05 14:08:12.647545320 +0100
@@ -33,7 +33,7 @@
 	Song& m_song;
 	std::stringstream m_ss;
 	unsigned int m_linenum;
-	bool getline(std::string& line) { ++m_linenum; return std::getline(m_ss, line);}
+	bool getline(std::string& line) { ++m_linenum; return static_cast<bool>(std::getline(m_ss, line));}
 	bool m_relative;
 	double m_gap;
 	double m_bpm;
