--- Performous-0.6.1-Source/cmake/Modules/FindGlib.cmake~	2010-10-31 17:05:44.000000000 +0100
+++ Performous-0.6.1-Source/cmake/Modules/FindGlib.cmake	2012-07-11 23:22:12.095571956 +0200
@@ -17,7 +17,7 @@
 find_path(GlibConfig_INCLUDE_DIR
   NAMES glibconfig.h
   PATHS ${Glib_PKGCONF_INCLUDE_DIRS} /usr
-  PATH_SUFFIXES lib/glib-2.0/include ../lib/glib-2.0/include
+  PATH_SUFFIXES lib${LIB_SUFFIX}/glib-2.0/include ../lib${LIB_SUFFIX}/glib-2.0/include
 )
 
 # Finally the library itself
--- Performous-0.6.1-Source/cmake/Modules/FindGDK-PixBuf.cmake~	2010-10-31 17:05:44.000000000 +0100
+++ Performous-0.6.1-Source/cmake/Modules/FindGDK-PixBuf.cmake	2012-07-11 23:25:09.845565687 +0200
@@ -14,7 +14,7 @@
 find_path(GDK-PixBuf_INCLUDE_DIR
   NAMES gdk-pixbuf/gdk-pixbuf.h
   PATHS ${GDK-PixBuf_PKGCONF_INCLUDE_DIRS}
-  PATH_SUFFIXES gtk-2.0
+  PATH_SUFFIXES gdk-pixbuf-2.0
 )
 
 # Finally the library itself
--- Performous-0.6.1-Source/cmake/Modules/FindLibRSVG.cmake~	2010-10-31 17:05:44.000000000 +0100
+++ Performous-0.6.1-Source/cmake/Modules/FindLibRSVG.cmake	2012-07-11 23:26:57.155561898 +0200
@@ -18,7 +18,7 @@
 find_path(LibRSVG_INCLUDE_DIR
   NAMES librsvg/rsvg.h
   PATHS ${LibRSVG_PKGCONF_INCLUDE_DIRS}
-  PATH_SUFFIXES librsvg-2
+  PATH_SUFFIXES librsvg-2.0
 )
 
 # Finally the library itself
--- Performous-0.6.1-Source/cmake/Modules/FindSigC++.cmake~	2010-10-31 17:05:44.000000000 +0100
+++ Performous-0.6.1-Source/cmake/Modules/FindSigC++.cmake	2012-07-11 23:29:10.248890540 +0200
@@ -17,7 +17,7 @@
 find_path(SigC++Config_INCLUDE_DIR
   NAMES sigc++config.h
   PATHS ${SigC++_PKGCONF_INCLUDE_DIRS} /usr
-  PATH_SUFFIXES lib/sigc++-2.0/include ../lib/sigc++-2.0/include
+  PATH_SUFFIXES lib${LIB_SUFFIX}/sigc++-2.0/include ../lib${LIB_SUFFIX}/sigc++-2.0/include
 )
 
 # Finally the library itself
--- Performous-0.6.1-Source/cmake/Modules/FindGlibmm.cmake~	2010-10-31 17:05:44.000000000 +0100
+++ Performous-0.6.1-Source/cmake/Modules/FindGlibmm.cmake	2012-07-11 23:29:52.625555711 +0200
@@ -21,7 +21,7 @@
 find_path(GlibmmConfig_INCLUDE_DIR
   NAMES glibmmconfig.h
   PATHS ${Glibmm_PKGCONF_INCLUDE_DIRS} /usr
-  PATH_SUFFIXES lib/glibmm-2.4/include ../lib/glibmm-2.4/include
+  PATH_SUFFIXES lib${LIB_SUFFIX}/glibmm-2.4/include ../lib${LIB_SUFFIX}/glibmm-2.4/include
 )
 
 # Finally the library itself
--- Performous-0.6.1-Source/cmake/Modules/FindLibXML++.cmake~	2010-10-31 17:05:44.000000000 +0100
+++ Performous-0.6.1-Source/cmake/Modules/FindLibXML++.cmake	2012-07-11 23:30:33.782220925 +0200
@@ -21,7 +21,7 @@
 find_path(LibXML++Config_INCLUDE_DIR
   NAMES libxml++config.h
   PATHS ${LibXML++_PKGCONF_INCLUDE_DIRS} /usr
-  PATH_SUFFIXES lib/libxml++-2.6/include ../lib/libxml++-2.6/include
+  PATH_SUFFIXES lib${LIB_SUFFIX}/libxml++-2.6/include ../lib${LIB_SUFFIX}/libxml++-2.6/include
 )
 
 # Finally the library itself
