--- performous-1.3.0/cmake/Modules/FindCed.cmake.orig	2023-08-21 21:48:26.000000000 +0200
+++ performous-1.3.0/cmake/Modules/FindCed.cmake	2023-10-20 10:23:51.966127665 +0200
@@ -2,10 +2,4 @@
 
 set(Ced_GIT_VERSION "master")
 
-libfetch_git_pkg(Ced
-	REPOSITORY ${SELF_BUILT_GIT_BASE}/compact_enc_det.git
-	#https://github.com/google/compact_enc_det.git
-	REFERENCE  ${Ced_GIT_VERSION}
-	FIND_PATH  compact_enc_det/compact_enc_det.h
-)
 message(STATUS "Found Google CED ${Ced_VERSION}")
