--- performous-1.1/game/CMakeLists.txt.orig	2016-01-23 11:40:01.000000000 +0100
+++ performous-1.1/game/CMakeLists.txt	2019-10-11 20:39:21.212719974 +0200
@@ -79,6 +79,24 @@
 	add_definitions(${${lib}_DEFINITIONS})
 endforeach(lib)
 
+#
+# harfbuzz
+#
+PKG_CHECK_MODULES(PC_HARFBUZZ REQUIRED harfbuzz)
+SET(HARFBUZZ_DEFINITIONS ${PC_HARFBUZZ_CXXFLAGS_OTHER})
+FIND_PATH(
+        HARFBUZZ_INCLUDE_DIR hb.h
+        HINTS ${PC_HARFBUZZ_INCLUDEDIR} ${PC_HARFBUZZ_INCLUDE_DIRS}
+        PATH_SUFFIXES harfbuzz)
+
+FIND_LIBRARY(
+        HARFBUZZ_LIBRARY NAMES harfbuzz
+        HINTS ${PC_HARFBUZZ_LIBDIR} ${PC_HARFBUZZ_LIBRARY_DIRS})
+
+include_directories(${HARFBUZZ_INCLUDE_DIR})
+list(APPEND LIBS ${HARFBUZZ_LIBRARY})
+
+
 find_package(Gettext)
 if(Gettext_FOUND)
 	include_directories(${Gettext_INCLUDE_DIRS})
